cmd/prometheus-postgresql-adapter/main.go

```diff
@@ -155,6 +156,7 @@ func parseFlags() *config {
 	flag.DurationVar(&cfg.prometheusTimeout, "leader-election-pg-advisory-lock-prometheus-timeout", -1, "Adapter will resign if there are no requests from Prometheus within a given timeout (0 means no timeout). "+
 		"Note: make sure that only one Prometheus instance talks to the adapter. Timeout value should be co-related with Prometheus scrape interval but add enough `slack` to prevent random flips.")
 	flag.BoolVar(&cfg.restElection, "leader-election-rest", false, "Enable REST interface for the leader election")
+	flag.DurationVar(&cfg.lockTimeout, "pg-advisory-lock-timeout", 3*time.Second, "PG advisory lock connection timeout duration. Adapter will resign in case of a lost connection since the lock will be lost as well.")
```

Comment:
Nit: I'd recommend the following argument description: `An interval used by adapter leader to make sure we still hold the lock`